Dockerfile

```diff
@@ -32,7 +32,8 @@ RUN echo -e "#!/bin/bash\nif [[ \"$1\" == \"--version\" ]]; then echo 'fake 244
 
 COPY manifest /manifest
 # Freeze packages and overwrite with overrides when needed
-RUN source /manifest; if [ -n "${ARCHIVE_DATE}" ]; then echo "Server=https://archive.archlinux.org/repos/${ARCHIVE_DATE}/\$repo/os/\$arch" > /etc/pacman.d/mirrorlist; else reflector --verbose --age 2 --latest 20 --protocol https --sort rate --save /etc/pacman.d/mirrorlist; fi && \
+RUN source /manifest && \
```

Comment:
Why removing this? Makes the Dockerfile more versatile, making it work for both frozen and non-frozen archive date